src/content/docs/aws/services/s3tables.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: "S3 Tables"
+description: Get started with Amazon S3 Tables on LocalStack
+persistence: supported
```

Comment:
not sure whether persistence is supported, we don't run any test to verify that - @bentsku could you help here and clarify what is required to support persistence in AWS service?

Reply:
Sure! By default, persistence would be enabled for the "control plane" via the store (we have some magic in place to automatically pick up stores), and here the "data plane" is in S3 so it should work by default. 

You can test it with our persistence tests suite framework. I'll send you our internal docs 👍